### ISSUE TITLE
Support JPEG as source images for scanned resources

### ISSUE THREAD

**Issue by escowles** [pulibrary/figgy#871]

**Comment by tpendragon** [comment]
I think we should get an example file so we can convert these to JP2s and see if they're worth zoom/panning or if we should just embed them as static images inside the IIIF manifest.

**Comment by hackmastera** [comment]
this ticket seems like it needs more grooming

**Comment by kevinreiss** [comment]
Here is an example file from the project that brought this question up.
![milberg-example](https://user-images.githubusercontent.com/278825/37936936-82a55128-3125-11e8-84dc-91bd5dc96d33.jpg)

**Comment by escowles** [comment]
The resolution of the example JPEG is 1867x2801, so somewhat lower than most of our TIFFs, but still sufficient to benefit from pan/zoom in a viewer.  So I think we can proceed with treating these JPEGs just like we treat TIFF source images, and fixing anything that fails (Kakadu is the obvious thing that will fail, but there could be others).

@hackmastera Is that enough info to make this ticket workable?  Support for JPEG source images is the main thing preventing us from moving forward with ingesting materials for an exhibit that's a high priority for the curator.

**Comment by hackmastera** [comment]
I looked at this briefly but I'm unassigning myself to work on another which is unblocking file manager work.

I ingested the file and observed that it does not generate derivatives. I looked at the derivatives code and updated the spec for `valid?`, observed that there's no mime type. So I think the problem starts in characterization. Didn't look at characterization code yet.

@eliotjordan suggests that this may have worked in plum; not sure if anyone else can confirm or wants to provide any pointers here.
